pkg/client/caller.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"encoding/json"
```

Comment:
Remember that the order of imports should be like:

- core packages

- your packages

- Third party packages